test/e2e/util/util.go

```diff
@@ -444,6 +444,9 @@ func EnsureHCPContainersHaveResourceRequests(t *testing.T, ctx context.Context,
 			t.Fatalf("failed to list pods: %v", err)
 		}
 		for _, pod := range podList.Items {
+			if strings.Contains(pod.Name, "-catalog-rollout-") {
```

Comment:
Just a note for later, would it make sense to extract this into a constant set and add comments w/ links to OCP bugs? Presumably we want this check to be temporary

Reply:
Ideally yes, we could open a bug against OLM to make this change.

And yes, if we find, that there are many exclusions, which hopefully there won't be, we can factor this out into a string set or something.